tc/core/tc2halide.cc

```diff
@@ -843,7 +843,9 @@ HalideComponents translateDef(const lang::Def& def, bool throwWarnings) {
     using IRMutator2::visit;
```

Comment:
Please add a note to the documentation to explain why the "Temporary hack" is needed.
Alternatively, remove the hack by exploiting #448.